engine/src/main/java/io/zeebe/engine/processing/streamprocessor/writers/Writers.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.engine.processing.streamprocessor.writers;
+
+/** Convenience class to aggregate all the writers */
+public final class Writers {
+
+  private final TypedStreamWriter stream;
+  private final StateWriter state;
+  private final CommandResponseWriter response;
+
+  public Writers(
+      final TypedStreamWriter stream,
+      final StateWriter state,
+      final CommandResponseWriter response) {
+    this.stream = stream;
+    this.state = state;
+    this.response = response;
+  }
+
+  /** @return the writer, which is used by the processors to write (follow-up) commands */
+  public TypedCommandWriter command() {
```

Comment:
I would prefer to adhere to the `getCommandWriter()` naming convention. I don't feel strongly about it, though. So feel free to implement or ignore this name change (ditto for other methods here)

Reply:
I don't. Let's name things within their context. https://github.com/kettanaito/naming-cheatsheet#avoid-context-duplication. However, the getter prefix might be interesting to use here. WDYT?

Reply:
I looked into it, but `writers.getCommand()` to get the `commandWriter` seems strange. It's either `writers.command()` or `writers.getCommandWriter()`. I then prefer the first.